lib/vmpooler.rb

```diff
@@ -52,6 +53,13 @@ def self.config(filepath='vmpooler.yaml')
       parsed_config[:graphite]['prefix'] ||= 'vmpooler'
     end
 
+    # statsd is an addition and my not be present in YAML configuration
+    if parsed_config[:statsd]
+      if parsed_config[:statsd]['server']
+        parsed_config[:statsd]['prefix'] ||= 'vmpooler'
+      end
+    end
```

Comment:
The struggle is real, and not in a flippant internet meme "my latte doesn't have a perfect rose on top" sort of sense.